src/main/webapp/components/query-manager/query-manager.tsx

```diff
@@ -56,7 +56,7 @@ const QueryCard = (props: QueryCardProps) => {
       >
         <Actions attributes={attributes}>
           <EditAction onEdit={handleOpen} />
-          <DeleteAction />
+          <DeleteAction itemName="query" onDelete={onDelete} />
```

Comment:
Should we put this in the `MetacardInteractionsDropdown` similar to how DDF does it? When we add the other icons, it may look a little crowded in the actions area.

Reply:
We can move it when it gets too crowded. Do we know if we definitely want the icons to be exactly the same as they are in intrigue? It seems that icons like the play button wouldn't be as useful on there.

Reply:
Probably warrants a discussion with the ux team.